sqlx-macros/src/query/mod.rs

```diff
@@ -271,13 +271,19 @@ where
         record_tokens
     };
 
-    let ret_tokens = quote! {{
-        use sqlx::Arguments as _;
+    let ret_tokens = quote! {
+        #[allow(clippy::all)]
```

Comment:
Why did you add this extra one? The whole issue is that we can't return an outer attribute on a block as an expression as it triggers an unstable feature error. If Clippy still complains without this then we need to think of something else.

Reply:
Oh I thought that’s what you meant by « {
    #[allow(clippy:all)]
    {
        ...
    }
}« 

Sorry I’ll fix it up as soon as I get to the computer

Reply:
just pushed, sorry for the noise I didn't quite understand why we wanted to have 2 sets of brackets before.
Your playground example is crystal clear, thanks!

Reply:
Better, thanks.